wikitemplate-IPFS.md

```diff
@@ -85,56 +130,91 @@
-  - [ ] Verify (`Linux`) that opening a shell and typing `xdg-open ipfs://bafkreigcnxudvpojjfwncmauociy5q46zsq46oe66cxbyzie3imabuoege` and pressing `Enter` opens Brave and loads an HTML page with the word `PASS`.
+## Peers
 
-## Peers:
-- [ ] Prerequisites: local node launched and local gateway configured.
+- [ ] Prerequisites: local node launched and local gateway configured on two machines, locally networked (LAN, can be over Wi-Fi).
 
-  ### Adding
+   ### `Adding` (see https://github.com/brave/brave-browser/issues/15567#issuecomment-867983572 for full setup steps)
 
-  - [ ] Verify when you go to `brave://settings/ipfs/peers` and click on the `Add` button, it prompts you to enter a new peer-connection string. Confirm that entering an incorrect string yields `This name is not valid` upon clicking `Submit`. (Acceptable ones are only CIDs or something like `**/p2p/**` format.)
-  - [ ] Verify if a peer is added and node is started, it proposes to restart node to apply changes.
-  - [ ] Verify the local node is restarted by clicking `Restart` button; happen it shows error message and suggests to see more on diagnostic page.
+   - [ ] Verify when you go to `brave://settings/ipfs/peers` and click on the `Add` button, it prompts you to enter a new peer-connection string. Confirm that entering an incorrect string yields `This name is not valid` upon clicking `Submit`. (Acceptable ones are only CIDs or something like `**/p2p/**` format.)
+   - [ ] Verify if a peer is added and node is started, it proposes to restart node to apply changes.
+   - [ ] Verify the local node is restarted by clicking `Restart` button; happen it shows error message and suggests to see more on diagnostic page.
 
-  ### Removing
+   ### `Removing`
 
-  - [ ] Verify a peer can be removed by clicking on Trash icon in the peer line.
+   - [ ] Verify a peer can be removed by clicking on the Trash icon in the peer line.
 
-## IPNS keys:
+## IPNS Keys
 
 - [ ] Prerequisites: local node launched and local gateway configured. Go to `Settings -> IPFS`, there should be an available `Set up your IPNS keys` option, which opens `brave://settings/ipfs/keys`
 
-  ### Sharing a local file using an IPFS key
+   ### `Sharing a local file using an IPFS key`
+
+   - [ ] Verify you're able to share a local file via `IPFS > Share local file using IPFS`.
+   - [ ] Verify once the file is imported, the import folder is opened and the file can be downloaded.
+   - [ ] Once the import is successful, verify a shareable link is copied to the clipboard by opening a new tab and pasting from the context menu or `CTRL+V`.
+   - [ ] Verify you see your key before the `?filename=filename.ext` in the copied text, e.g. `https://dweb.link/ipns/k51qzi5uqu5dgxhiv8w8cdvmgdhbvy3t9gn4jwpwwro18fots0xtdabpcxxzwc?filename=Big_Buck_Bunny_4K.webm` (the key is `k51qzi5uqu5dgxhiv8w8cdvmgdhbvy3t9gn4jwpwwro18fots0xtdabpcxxzwc`).
```

Comment:
Can shorten this one to 

- [ ] Verify you see the IPFS key(`k51qzi5uqu5dgxhiv8w8cdvmgdhbvy3t9gn4jwpwwro18fots0xtdabpcxxzwc`) before the `?filename=filename.ext` from the copied text.

Reply:
Fixed.